<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_addtacheunaire</class>
 <widget class="QWidget" name="window_addtacheunaire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations principales</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Titre</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="t_titre"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Echéance</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateTimeEdit" name="dt_ech"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Disponibilité</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateTimeEdit" name="dt_dispo"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QCheckBox" name="cb_prereq">
      <property name="text">
       <string>prérequis ?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Type de tache</string>
   </property>
   <widget class="QRadioButton" name="rb_unaire">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Unaire</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_preemp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Préemptable</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_unaire">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations tache unaire</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Durée (en heures)</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="t_duree">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumTime">
     <time>
      <hour>12</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_prereq">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Gestion des tâches prérequises</string>
   </property>
   <widget class="QListView" name="list_prereq">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sélectionner les prérequis</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="b_ok">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
